environment/templates/fabric8-tenant-jenkins.yml

```diff
@@ -834,11 +834,46 @@ objects:
     to:
       kind: Service
       name: jenkins
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    labels:
+      app: jenkins
+      provider: fabric8
+      version: ${COMMIT}
+      version-quotas: ${COMMIT_QUOTAS}
+    name: openshift-jenkins-login-plugin-config
+    namespace: ${USER_NAME}-jenkins
+  data:
+    Overall-Read: admin,edit,view
+    Overall-RunScripts: admin,edit,view
+    Overall-Administer: admin
+    Job-Build: admin,edit,view
+    Job-Configure: admin,edit,view
+    Job-Create: admin,edit,view
+    Job-Delete: admin,edit,view
+    Job-Cancel: admin,edit,view
+    Job-Wokspace: admin,edit,view
+    Job-Read: admin,edit,view
+    Job-Discover: admin,edit,view
+    Run-Delete: admin
+    Run-Update: admin
+    View-Configure: admin
+    View-Create: admin
+    View-Delete: admin
+    Credentials-Create: admin
+    Credentials-Update: admin
+    Credentials-Delete: admin
+    Credentials-ManageDomains: admin
+    Credentials-View: admin,edit,view
+    SCM-Tag: admin,edit,view
+    Slave-Configure: admin
+    Slave-Delete: admin
 parameters:
 - name: USER_NAME
   value: developer
 - name: JENKINS_OPENSHIFT_VERSION
-  value: vb355188
+  value: SNAPSHOT-PR-234-140
```

Comment:
Do we need to update it before merge?

Reply:
Yes, we need to first merge PR on config repo. That will send an automated PR for image tag update after that we need to remove this part and merge.